server/model/monitor.js

```diff
@@ -383,6 +383,8 @@ class Monitor extends BeanModel {
                     bean.msg = "Monitor under maintenance";
                     bean.status = MAINTENANCE;
                 } else if (this.type === "group") {
+                    let failChild = "";
```

Comment:
I think it would be better to collect the `DOWN`-children in a list instead of doing this double accounting (last name + count).
This would allow adding this to the logging output, and I think the code might look better.

Reply:
Even including the count was beyond the scope of the original request. It's simple, doesn't use any extra memory, and makes the message more informative. I'll stick with what I have.

Reply:
I am not convinced that the extra memory of a few child-names will matter for overall performance/memory consumption.
Yes, my approach does consume a bit more memory, but assuming reasonably long names and a reasonable ammount of monitors per group.
I do think adding a separate mechanism for accounting for the failed number is sort of a footgun => I would prefer a list for tracking this ^^